.github/workflows/ci.yml

```diff
@@ -30,15 +30,17 @@ jobs:
 
     - name: Install
       run: |
-        pip install --upgrade pytest pytest-mock codecov pytest-cov h5py pyyaml mockmpi
-        pip install dask[distributed]
+        sudo apt-get update && sudo apt-get -y install libopenmpi-dev openmpi-bin graphviz graphviz-dev
+        pip install --upgrade pip wheel
+        pip install .[all] pytest pytest-mock codecov pytest-cov h5py pyyaml mockmpi dask[distributed]
         pip install --no-deps git+https://github.com/joezuntz/dask-mpi
```

Comment:
This is getting a bit complicated to just live in the unit tests.   Maybe collect the extras into additional build targets.

Reply:
Done in the new pyproject.toml.  Let's see if it works.

Reply:
It did, eventually!